feature/mpls/sr/otg_tests/srte_egress/egress_node_forwarding /README.md

```diff
@@ -0,0 +1,190 @@
+# SR-1.2: Egress Node Forwarding
+
+## Summary
+
+This is to test the forwarding functionality of MPLS traffic on an Egress node 
+in SRTE+MPLS enabled network.
```

Comment:
(nit) Typically, we write this as "SR-MPLS" since the original SR RFCs. Suggest to stick with this, since SRTE often means "SR-TE policy".